extra-devel/sandsifter/autobuild/patches/0001-fix-build-with-glibc-2.34.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/injector.c b/injector.c
```

Comment:
Whose patch is this? Follow our commit convention:

```
sandsifter: (whose patch?) ...

- What did it fix and how?

Ref: ...
```

Reply:
This is the original patch. The co-author helps me create this patch.